src/VirtualClient/VirtualClient.Main/profiles/QUAL-GPU-DCGMI-DIAG.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "Description": "DCGMI Monitor",
+  "Parameters": {
+    "Level": "4",
+    "Username": "testuser",
+    "CudaVersion": "11.6",
+    "DriverVersion": "510",
+    "LocalRunFile": "https://developer.download.nvidia.com/compute/cuda/11.6.0/local_installers/cuda_11.6.0_510.39.01_linux.run"
+  },
+  "Dependencies": [
+    {
+      "Type": "CudaAndNvidiaGPUDriverInstallation",
+      "Parameters": {
+        "Scenario": "InstallCudaAndNvidiaGPUDriver",
+        "CudaVersion": "$.Parameters.CudaVersion",
+        "DriverVersion": "$.Parameters.DriverVersion",
+        "Username": "$.Parameters.Username",
+        "LocalRunFile": "$.Parameters.LocalRunFile"
+      }
+    },
+    {
+      "Type": "DCGMIInstallation",
+      "Parameters": {
+        "Scenario": "InstallDCGMI"
+      }
+    }
+  ],
+  "Actions": [
```

Comment:
Dependencies ALWAYS at the bottom so that we are consistent with the rest of our profiles.